packages/ses/src/whitelist.js

```diff
@@ -737,7 +737,7 @@ export default {
     indexOf: fn,
     // 21.1.3.9 String.prototype.lastIndexOf
     lastIndexOf: fn,
-    // 21.1.3.10 String.prototype.localeCompare suppressed
+    // 21.1.3.10 String.prototype.localeCompare
     localeCompare: false,
```

Comment:
BUG BUG BUG. This should have been changed from `false` to `fn`. With it as `false`, `localeCompare` was still being removed. Amusing but sad why the tests passed anyway. Fixing both now.